SvrfSDK/Source/SvrfAPIManager.swift

```
@@ -0,0 +1,180 @@
+//
+//  SvrfAPIManager.swift
+//  SvrfSDK
+//
+//  Created by Andrei Evstratenko on 28/05/2019.
+//  Copyright © 2019 Svrf, Inc. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+class SvrfAPIManager {
+
+    static private let baseURL = "https://api.svrf.com/v1"
+    static private var xAppToken: String?
+
+    static func setToken(_ token: String) {
+        xAppToken = token
+    }
+
+    static func authenticate(with apiKey: String,
+                             onSuccess success: @escaping (_ authenticationResponse: SvrfAuthenticationResponse) -> Void,
+                             onFailure failure: @escaping (_ error: Error?) -> Void) {
+
+        let path = "/app/authenticate"
+        let urlString = baseURL + path
+
+        let header = ["content-type": "application/json"]
+        let parameters = ["apiKey": apiKey]
+
+        Alamofire.request(urlString,
+                          method: .post,
+                          parameters: parameters,
+                          encoding: JSONEncoding.default,
+                          headers: header).responseJSON { response in
+
+            if let jsonData = response.data {
+                do {
+                    let decoder = JSONDecoder()
+                    let authenticationResponse = try decoder.decode(SvrfAuthenticationResponse.self, from: jsonData)
+
+                    success(authenticationResponse)
+                } catch let error {
+                    failure(error)
+                }
+            }
+        }
+    }
+
+    static func getMedia(by identifier: String,
+                         onSuccess success: @escaping (_ mediaResponse: SvrfMediaResponse) -> Void,
+                         onFailure failure: @escaping (_ error: Error?) -> Void) {
+
+        let path = "/vr/\(identifier)"
+        let urlString = baseURL + path
+
+        let header = ["x-app-token": xAppToken ?? "",
```

Is `xAppToken` a required header for these calls? If so, should we fail immediately here if it's not set?

---

Added check for x-app-token